XamlControlsGallery/ControlPages/ToggleButtonPage.xaml.cs

```diff
@@ -36,5 +36,14 @@ private void ToggleButton_Unchecked(object sender, RoutedEventArgs e)
         {
             Control1Output.Text = "Off";
         }
+        private void ToggleButtonReveal_Checked(object sender, RoutedEventArgs e)
+        {
+            ControlRevealOutput.Text = "On";
+        }
+
+        private void ToggleButtonReveal_Unchecked(object sender, RoutedEventArgs e)
+        {
+            ControlRevealOutput.Text = "Off";
```

Comment:
The other toggle button loads with the "Off" text visible. Can you update this one to do the same?

Reply:
Fixed with latest commit.